infra_tooling_account/infra_tooling_account/infra_tooling_alerting_stack.py

```diff
@@ -45,25 +51,40 @@ def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
             event_bus_name=f"eventbus-{project_name}-notification-{stage_name}"
         )
 
+        # Notification Lambda Eventbridge rule
+        notification_lambda_event_rule = events.Rule(
```

Comment:
1. I believe this rule should be in monitored account (where events are generated)
2. let's make pattern more specific
  aws:glue, ... <- services we CAN handle (+ step functions, + (?) lambda if it's possible, + (?) glue workflow if it's separate from glue itsleft)

Reply:
Discussed - leaving as it should be